docs/SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please contact the Security Team: `cncf-kubeedge-security@lists.cncf.io`. Security is of the highest importance and we sincerely request you to keep the vulnerability information confidential.
+
+The team will help diagnose the severity of the issue and determine how to address the issue. The reporter(s) can expect a response within 24 hours acknowledging the issue was received. If a response is not received within 24 hours, please reach out to any committer directly to confirm receipt of the issue.
```

Comment:
```suggestion
The team will help diagnose the severity of the issue and determine how to address the issue. The reporter(s) can expect a response within 24 hours acknowledging the issue was received. If a response is not received within 24 hours, please reach out to any Security Team member directly to confirm receipt of the issue.
```

Should be Security Team member, instead of committer here.